docs/build-process/schema-descriptor-build-process.md

```diff
@@ -0,0 +1,19 @@
+# Building a schema descriptor
+This page explains how to build a schema descriptor in a SpatialOS project that’s using the flexible project layout (FPL).
+
+## Schema descriptor introduction 
+The schema descriptor is a build artifact of a SpatialOS project [assembly](https://docs.improbable.io/reference/13.6/shared/glossary#assembly). It allows the SpatialOS Runtime to interpret your project’s schema and is necessary for starting a SpatialOS deployment both locally and in the cloud. You can generate a schema descriptor using the [schema compiler tool](https://docs.improbable.io/reference/latest/shared/schema/introduction#using-the-schema-compiler-directly).
+
+**Note:** In the [current project layout (CPL)](https://docs.improbable.io/reference/latest/shared/reference/project-structure), the spatial CLI automatically generates the schema descriptor. For an FPL project, you need to manually generate your schema descriptor, as you now fully manage the build process of your project.
```

Comment:
"For an FPL project, you need to manually generate your schema descriptor, as you now fully manage the build process of your project."

Sounds a bit strong to me. Are there any benefits of generating schema descriptor yourself vs. automatically before?

Reply:
This is an attempt to make something bad sound good. ;)
The full context behind this is: In the olde project layout, we did a lot of stuff under the hood for you when you built your project. The build process was also more opinionated. In FPL, the build process is less opinionated and you have more control over it but you need to build everything yourself as a tradeoff. I will remove the last part of the sentence.